src/api/endpoints/activity_types.py

```diff
@@ -96,6 +98,8 @@ def put(cls, act_types_id=None):
             target_activity_type.description = payload.get("description")
         if payload.get("value"):
             target_activity_type.value = payload.get("value")
+        if payload.get("supports_multiple_participant"):
```

Comment:
The long name will cause errors in future such as how the name already has differences with what in the lines below it.

Reply:
@Andretalik, thanks for the feedback. Names should reflect what the variable is and hopefully what it does. Which name do you suggest?